lte/gateway/c/session_manager/LocalEnforcer.cpp

```diff
@@ -254,8 +254,8 @@ void LocalEnforcer::aggregate_records(
     SessionStateUpdateCriteria& uc = session_update[imsi][session_id];
 
     session->add_rule_usage(
```

Comment:
maybe we should just pass in the full record here.

Reply:
there are so many test cases to change tho

Reply:
yeah true maybe add it as a todo in the header